src/styles/brackets.less

```diff
@@ -1207,12 +1207,23 @@ a, img {
     }
 
     .close {
+        background-image: url("../extensions/default/bramble/images/close-icon-dark.svg");
```

Comment:
If we're going to use this icon for code outside the `extensions/` dir, we are going to want to move that file, and update all users of it.  Probably this needs to live in `src/styles/images/`.  It's not a good idea for us to depend on URLs under `extensions` here, since that stuff can change/move.